ui/app/components/lifecycle-chart-row.js

```diff
@@ -11,13 +11,45 @@ import classic from 'ember-classic-decorator';
 @classic
 @tagName('')
 export default class LifecycleChartRow extends Component {
-  @computed('taskState.state')
-  get activeClass() {
-    if (this.taskState && this.taskState.state === 'running') {
-      return 'is-active';
+  get taskColor() {
+    let color;
+    if (this.taskState?.state === 'running') {
+      color = 'success';
+    }
+    if (this.taskState?.state === 'pending') {
+      color = 'neutral';
+    }
+    if (this.taskState?.state === 'dead') {
+      if (this.taskState?.failed) {
+        color = 'critical';
+      } else {
+        color = 'neutral';
+      }
     }
+    return color;
```

Comment:
What happens if this returns undefined?  Does it just use some default color?

Reply:
That's a good point; in this case it would cause an error because the component requires one of a small set of strings. I'll set a default on it.